hm_fashion_recs/week_processing_benchmark.py

```diff
@@ -0,0 +1,118 @@
+from hm_fashion_recs.fe import attach_features
+
+
+from hm_fashion_recs.vars import (
+    raw_data_path,
+    preprocessed_data_path,
+    user_features_path,
+    lfm_features_path,
+    working_dir,
+)
+from hm_fashion_recs.tm import tm
+
+
+from utils import check_support, trigger_execution, Config
+
+
+class CFG:
+    raw_data_path = raw_data_path
+    preprocessed_data_path = preprocessed_data_path
+    lfm_features_path = lfm_features_path
+    working_dir = working_dir
+    user_features_path = user_features_path
+
+    use_lfm = False
+
+
+def feature_engieering(week):
+    with tm.timeit("01-load_data"):
+        transactions, users, items = load_data()
+
+    with tm.timeit("02-age_shifts"):
+        age_shifts = get_age_shifts(transactions=transactions, users=users)
+
+    with tm.timeit("03-candidates"):
+        week_candidates = make_one_week_candidates(
+            transactions=transactions,
+            users=users,
+            items=items,
+            week=week,
+            user_features_path=CFG.user_features_path,
+            age_shifts=age_shifts,
+        )
+
+        candidates = drop_trivial_users(week_candidates)
+
+        candidates.to_pickle(CFG.working_dir / "candidates.pkl")
+
+    with tm.timeit("04-attach_features"):
+        dataset = attach_features(
+            transactions=transactions,
+            users=users,
+            items=items,
+            candidates=candidates,
+            # +1 because train data comes one week earlier
+            week=week + 1,
+            pretrain_week=week + 2,
+            age_shifts=age_shifts,
+            user_features_path=CFG.user_features_path,
+            lfm_features_path=CFG.lfm_features_path if CFG.use_lfm else None,
+        )
+
+    dataset["query_group"] = dataset["week"].astype(str) + "_" + dataset["user"].astype(str)
+    dataset = dataset.sort_values(by="query_group").reset_index(drop=True)
+    return dataset
+
+
+def main(raw_data_path):
+    with tm.timeit("total"):
+        with tm.timeit("01-initial_transform"):
+            transform_data(input_data_path=raw_data_path, result_path=CFG.preprocessed_data_path)
+
+        week = 0
+        with tm.timeit("02-create_user_ohe_agg"):
+            create_user_ohe_agg(
+                week + 1,
+                preprocessed_data_path=CFG.preprocessed_data_path,
+                result_path=CFG.user_features_path,
+            )
+
+        with tm.timeit("03-fe"):
+            feature_engieering(week=week)
+
+    print(tm.get_results())
+
+
+def run_benchmark(parameters):
+    check_support(parameters, unsupported_params=["optimizer", "dfiles_num"])
+
+    # Update config in case some envs changed after the import
+    Config.init(
+        MODIN_IMPL="pandas" if parameters["pandas_mode"] == "Pandas" else "modin",
+        MODIN_STORAGE_FORMAT=os.getenv("MODIN_STORAGE_FORMAT"),
+        MODIN_ENGINE=os.getenv("MODIN_ENGINE"),
```

Comment:
```suggestion
        MODIN_STORAGE_FORMAT=cfg.StorageFormat.get(),
        MODIN_ENGINE=cfg.Engine.get(),
```
This requires the env vars to be set beforehand. Otherwise, they are None. I suggest using the respective configs, which have the valid default values.

Reply:
I don't mind doing this in a separate PR but even now I do not see the initialization of `os.environ["MODIN_STORAGE_FORMAT"] = "pandas"` in case of engines other than hdk.

Reply:
Root cause of the issue is solved in https://github.com/intel-ai/omniscripts/pull/341